float32 processing_time
int16[] path_x
int16[] path_y
float32 path_heading